webui/react/src/contexts/ClusterOverview.tsx

```diff
@@ -19,6 +19,7 @@ type Action =
 const defaultClusterOverview: ClusterOverview = {
   [ResourceType.CPU]: { available: 0, total: 0 },
   [ResourceType.GPU]: { available: 0, total: 0 },
+  [ResourceType.UNSPECIFIED]: { available: 0, total: 0 },
```

Comment:
This change has visible consequences in the cluster page. Should UNDEFINED be ignored?

Reply:
Yes let's ignore `UNSPECIFIED`

Reply:
Done.